mpf/devices/multiball_lock.py

```diff
@@ -144,14 +144,16 @@ def locked_balls(self, value):
                 self.config['locked_ball_counting_strategy']))
 
     def _register_handlers(self):
+        priority = (self.mode and self.mode.priority or 0) + \
```

Comment:
Nit: While `(self.mode and self.mode.priority or 0)` is correct I would use: `(self.mode.priority if self.mode else 0) ` to make it more clear what is happening here